### PR TITLE
PIN-798 added contract data to token

### PR DESCRIPTION
**Pull request by ubaldop**

### CODE REVIEW

File: src/main/protobuf/v1/token.proto [hunk 1]
```diff
@@ -11,6 +11,8 @@ message TokenV1 {
   repeated PartyRelationshipBindingV1 legals = 2;
   required string validity = 3;
   required string checksum = 4;
+  required string contractPath = 5;
+  required string contractVersion = 6;
```
Comment (galales) [inline src/main/protobuf/v1/token.proto]: may it have sense to group these fields under the one object?

File: src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/api/impl/package.scala [hunk 2]
```diff
@@ -20,7 +20,7 @@ package object impl extends SprayJsonSupport with DefaultJsonProtocol {
   implicit val relationshipsFormat: RootJsonFormat[Relationships]         = jsonFormat1(Relationships)
   implicit val relationshipsSeedFormat: RootJsonFormat[RelationshipsSeed] = jsonFormat1(RelationshipsSeed)
   implicit val problemFormat: RootJsonFormat[Problem]                     = jsonFormat3(Problem)
-  implicit val tokenFeedFormat: RootJsonFormat[TokenSeed]                 = jsonFormat3(TokenSeed)
+  implicit val tokenFeedFormat: RootJsonFormat[TokenSeed]                 = jsonFormat5(TokenSeed)
```
Comment (beetlecrunch) [inline src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/api/impl/package.scala]:
```suggestion
  implicit val tokenSeedFormat: RootJsonFormat[TokenSeed]                 = jsonFormat5(TokenSeed)
```

**Review by galales — APPROVED** [review]

File: src/main/protobuf/v1/token.proto (hunk 1 above)
Comment (ubaldop) [reply src/main/protobuf/v1/token.proto]: please, consider that these become token claims. Could be annoying to have nested ones?

**Review by beetlecrunch — APPROVED** [review]

File: src/main/protobuf/v1/token.proto (hunk 1 above)
Comment (galales) [reply src/main/protobuf/v1/token.proto]: not sure, annoying referred to which issue in particular?

Comment (ubaldop) [reply src/main/protobuf/v1/token.proto]: I don't know, I use to see flat jwt claims 🤷🏻‍♂️

**Review by galales — APPROVED** [review]

**Pull request merged**